src/components/CommentsForm.jsx

```diff
@@ -0,0 +1,100 @@
+import {  useState } from "react";
+import { useParams } from "react-router-dom";
+import { postComments } from "../utils/api";
+
+
+
+const CommentsForm =({commentsList, setCommentsList,})=>{
+const params =useParams();
+
+const articleId = params.article_id;
+
+const[userName, setUserName]=useState('tickle122')
+
+const[body, setBody]=useState("")
+const [err, setErr] =useState(null);
+
+const[message, setMessage]= useState("");
+
+
+
+const handleSubmit =(event)=>{
+    event.preventDefault();
+   
+    setUserName("tickle122");
```

Comment:
There's no point at which you change this state so you don't need this line.
Strictly as this isn't changed anywhere it doesn't need to be in a state 🤔